packages/vaadin-overlay/src/vaadin-overlay-position-mixin.js

```diff
@@ -196,28 +211,26 @@ export const _PositionMixin = (superClass) =>
       const shouldGoToDefaultSide =
         spaceForDefaultAlignment > spaceForOtherAlignment || spaceForDefaultAlignment > contentSize;
 
-      const shouldAlignStart = defaultAlignStart === shouldGoToDefaultSide;
+      return defaultAlignStart === shouldGoToDefaultSide;
+    }
 
+    /**
+     * Returns an object with CSS position properties to set,
+     * e.g. { top: "100px", bottom: "" }
+     */
+    static __calculatePositionInOneDimension(targetRect, overlayRect, noOverlap, propNames, overlay, shouldAlignStart) {
```

Comment:
Does it make sense to keep this a static class method? Maybe we could make it a function?